espnet/nets/pytorch_backend/transformer/encoder_layer.py

```diff
@@ -71,7 +71,6 @@ def forward(self, x, mask, cache=None):
             assert cache.shape == (x.shape[0], x.shape[1] - 1, self.size)
             x_q = x[:, -1:, :]
             residual = residual[:, -1:, :]
-            mask = None if mask is None else mask[:, -1:, :]
```

Comment:
Could you explain this change as well?

Reply:
When mask shape is [batch, 1, max_length], mask[:, -1:, :] is same with mask. I did not find the case that mask.shape[1] != 1 in the code. Please correct me if I am wrong. Thanks!

Reply:
Sorry I just find the mask is also used in LM transformer part. I will read the code thoroughly and then refine the comments. close this PR now.

Reply:
OK, but you can make a separate PR to just fix the documentation part.